logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/keypackage/MLSKeyPackageCountUseCaseTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.wire.kalium.logic.feature.keypackage
+
+import com.wire.kalium.logic.CoreFailure
+import com.wire.kalium.logic.NetworkFailure
+import com.wire.kalium.logic.data.client.ClientRepository
+import com.wire.kalium.logic.data.keypackage.KeyPackageRepository
+import com.wire.kalium.logic.framework.TestClient
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.network.api.keypackage.KeyPackageCountDTO
+import io.mockative.Mock
+import io.mockative.anything
+import io.mockative.classOf
+import io.mockative.given
+import io.mockative.mock
+import io.mockative.once
+import io.mockative.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertIs
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class MLSKeyPackageCountUseCaseTest {
```

Comment:
I'm not sure if it's necessary in this case but I think we agreed to create new test classes using "arrangement" pattern 🤔

Reply:
Hmm! Right, but it's small test class. I think it's not necessary here.